### ISSUE TITLE
🚸 Seeing ahead of time which features I'll get

### ISSUE THREAD

**Issue by falexwolf** [laminlabs/lamindb#2281]
I'm very surprised that I only have two obs features in the annotation:

https://lamin.ai/laminlabs/lamindata/transform/7Q2GS36KPgX90000

<img width="834" alt="image" src="https://github.com/user-attachments/assets/481cbd8f-a630-4f29-9158-8309bcc22fab" />

Is there a way to display what I'll get before hitting `save_artifact()`?

**Comment by Zethson** [comment]
No, the `Artifact` will only be created at the `save_artifact()` stage. This is bugging me a tiny bit as well in certain situations.

**Comment by sunnyosun** [comment]
You can if you turn on hint verbosity, it will show what is validated when you run `.validate()`.
